### PR TITLE
Add @react-native-community/[picker|slider]

### PR DESCRIPTION
**Pull request by brentvatne**
# Why

They're on a deprecation path from core and they are very useful components!

# How

Added import script, imported them, added docs pages, updated NCL.

# Test Plan

Test them in NCL:
- [x] iOS
- [x] Android
- [x] Web

# TODO

- [x] ~Fix NCL on web - need to make sure we import Picker from react-native always on web. Bonus: open PR on Picker so it imports from react-native on web until they implement this.~ @EvanBacon reviewed and landed a PR upstream for Picker so we are good to go here.
- [ ] Follow up on slider 3.0.0 proper being released and update this PR to use it.

### CODE REVIEW

File: packages/expo/bundledNativeModules.json
Comment (ide) [inline packages/expo/bundledNativeModules.json:102]: Will this make its way from RC to a "final" version in the next couple of days? Otherwise unless there are serious issues with 2.x I think we should prefer the stable version.

Comment (brentvatne) [reply packages/expo/bundledNativeModules.json:102]: yeah i would go for 2.x if not. i have pinged the owners of this library on discord a couple times. my concern if we go with 2.x is that it doesn't have web support that @EvanBacon added, so i'm hoping that they will land 3.0 proper soon.

**Review by ide — APPROVED** [review]
Approving contingent on 3.x being ready (or at least committing to only changes to the public JS API).

**Review by tsapeta — APPROVED** [review]

**Comment by brentvatne** [comment]
**Update:** there were a couple of pending PRs that slider maintainers wanted to land before releasing, I pushed those through so hopefully on Tuesday we will be able to merge this.

**Comment by brentvatne** [comment]
Merging with intent to update to @react-native-community/slider@3.0.0 asap

**Pull request merged**
